### ISSUE TITLE
Notification opens the wrong direct

### ISSUE THREAD

**Issue by SerPenTargo** [RocketChat/Rocket.Chat#30455]
### Search before asking

- [X] I had searched in the [issues](https://github.com/RocketChat/Rocket.Chat.Electron/issues?q=is%3Aissue) and found no similar issues.

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

### Operating System Version

Windows 11 Pro 22H2 22621.2283

### It happens on the web browser too?

No, it just happens on the Desktop app

### Rocket.Chat Desktop App Version

3.9.7

### Rocket.Chat Server Version

6.3.7

### Describe the bug

Clicking on a notification opens not the direct chat that was the source of the notification, but another chat last opened in the RC window.

In other words:
I've been text with John.
I minimized the RC window.
After a while, a message from David comes in and a notification.
I click on the notification.
The direct with John, not David, opens.

### How to Reproduce

1. Minimize RC with user 1.
2. Wait for notification from user 2.
3. Click on it.

### Describe your Expected behavior

The direct window opens with user 2.

### Are you willing to submit a code contribution?

- [ ] Yes, I am willing to submit a Pull Request!

**Comment by jeanfbrito** [comment]
This is happening on the browser too. Seems to be a server regression.

**Comment by Lancer854** [comment]
The same problem. Rocket.Chat Desktop App Version 3.9.8

**Comment by Canepro** [comment]
\nI confirmed this as reproducible from the report and marked it as triaged. This looks customer-impacting, so I am escalating this to CSB Engineering for further review.

**Comment by Canepro** [comment]
I confirmed this as reproducible from the report and marked it as triaged. This appears customer-impacting, so I am escalating this to CSB Engineering for further review.
